example.env

```
@@ -33,6 +33,9 @@ FACEBOOK_SECRET=secret_for_facebook_oauth
 GOOGLE_OAUTH2_CLIENT_ID=client_id_for_google_oauth
 GOOGLE_OAUTH2_CLIENT_SECRET=client_secret_for_google_oauth
 
+# Default sender email address
+DEFAULT_SENDER_EMAIL=noreply@pupilfirst.com
```

```suggestion
DEFAULT_SENDER_ADDRESS=noreply@example.com
```

...or `DEFAULT_SEND_EMAIL_ADDRESS`. The word _email_, alone, has a different meaning.